### PR TITLE
my story PRJ

### PR DESCRIPTION
**Pull request by princeraj14**
Please Accept

### CODE REVIEW

File: prince.story [hunk 1]
```diff
@@ -0,0 +1 @@
+Prince Raj Loves Banana And Grapes
```
Comment (gopalgoel) [inline prince.story]: :wink:  Very Good Commit

Comment (princeraj14) [inline prince.story]: Thanks

Comment (rishabhraj9) [inline prince.story]: good job keep loving them

Comment (dextar101010) [inline prince.story]: [www.youtube.com](url)

Comment (nishu707) [inline prince.story]: change grapes to babes

**Pull request merged**